README.md

```diff
@@ -0,0 +1,57 @@
+# Central Website for Students’ Gymkhana​
+
+## An Initiative by Gymkhana Web Team
+
```

Comment:
Make the details here more technical like how we are going to use a single frontend and backend to make webpages of multiple club, how we provide customize options